shared/src/common/enums/sort/sort.ts

```diff
@@ -0,0 +1,5 @@
+enum SortOrder {
```

Comment:
I think Vlad meant you to take `SortOrder` enum inside `sort-order.enum` file which is inside `enums/sort` folder, then it's being exported through root `sort.ts` file
eg
inside `enums/sort` you have 2 files
```
sort-order.enum.ts // with enum declared inside
sort.ts // with export { SortOrder } from './sort-order.enum'
```
and then as all things stuff from `enums/sort` is also exported from root `enums` file

Reply:
Oh, thank you!